app/src/main/java/com/github/se/icebreakrr/ui/profile/InboxProfileView.kt

```diff
@@ -0,0 +1,241 @@
+
+private const val ACCEPT_DECLINE_ELEVATION = 4
+private const val ACCEPT_DECLINE_ROUNDER_CORNER_SHAPE = 23
+private const val ACCEPT_DECLINE_SHADOW_COLOR = 0x40000000
+private const val ACCEPT_DECLINE_BOX_COLOR = 0xFFEAEEFF
+private const val ACCEPT_DECLINE_BOX_HEIGHT = 115
+private const val ACCEPT_DECLINE_BOX_WIDTH_FACTOR = 0.9
+private const val ACCEPT_DECLINE_PADDING_START = 10.5
+private const val ACCEPT_DECLINE_PADDING_END = 14.5
+private const val ACCEPT_DECLINE_PADDING_TOP_BOTTOM = 8
+private const val REQUEST_MESSAGE_FONT_SIZE = 20
+private const val REQUEST_MESSAGE_LINE_HEIGHT = 20
+private const val REQUEST_MESSAGE_FONT_WEIGHT = 500
+private const val REQUEST_MESSAGE_LETTER_SPACING = 0.2
+private const val SPACER_HEIGHT = 10
+private const val ACCEPT_DECLINE_ICON_BUTTON_PADDING = 12
+private const val ACCEPT_DECLINE_ICON_BUTTON_SIZE = 48
+private const val ACCEPT_DECLINE_ICON_BUTTON_ROUNDED = 100
+private const val ACCEPT_DECLINE_ICON_BUTTON_COLOR = 0xFF65558F
+
+/**
+ * Screen that appears when you click on a profile in the notification tab. You can accept or
+ * decline the notification.
+ *
+ * @param profilesViewModel : view model of the profile
+ * @param navBackStackEntry : used to retrieve the uid of the user that we clicked on
+ * @param navigationActions : go back when we accept, decline or go back
+ * @param tagsViewModel : used to display user's tag
+ * @param meetingRequestViewModel : view model to send/receive meeting requests
+ * @param isTesting : indicates if we are in testing mode or not
+ */
+@Composable
+fun InboxProfileViewScreen(
+    profilesViewModel: ProfilesViewModel,
+    navBackStackEntry: NavBackStackEntry?,
+    navigationActions: NavigationActions,
+    tagsViewModel: TagsViewModel,
+    meetingRequestViewModel: MeetingRequestViewModel,
+    isTesting: Boolean
+) {
+
+  LaunchedEffect(Unit) {
+    val profileId = navBackStackEntry?.arguments?.getString("userId")
+    if (profileId != null || isTesting) {
+      profilesViewModel.getProfileByUid(profileId ?: "")
+      meetingRequestViewModel.updateInboxOfMessages()
+    }
+  }
+
+  val isLoading = profilesViewModel.loading.collectAsState().value
+  val profile = profilesViewModel.selectedProfile.collectAsState().value
+  val inboxItems = profilesViewModel.inboxItems.collectAsState()
+  val context = LocalContext.current
+
+  val message = inboxItems.value[profile]
+
+  Scaffold(modifier = Modifier.fillMaxSize().testTag("NotificationProfileScreen")) { paddingValues
+    ->
+    if (isLoading) {
+      MessageWhenLoadingProfile(paddingValues = paddingValues)
+    } else if (profile != null) {
+      Column(horizontalAlignment = Alignment.CenterHorizontally) {
+        ProfileHeader(
+            profile = profile,
+            navigationActions = navigationActions,
+            myProfile = false,
+            profilesViewModel = profilesViewModel,
+            profileInNotification = true,
+            onEditClick = {})
```

Comment:
Nitpick: If onEditClick is unused, consider passing a null lambda or omitting it to simplify the code.

Reply:
I changed the parameter to  `onEditClick: (() -> Unit)?` and I pass null to onEditClick.